src/main/java/org/example/fuzzy_matcher/util/DataReader.java

```diff
@@ -0,0 +1,42 @@
+package org.example.fuzzy_matcher.util;
```

Comment:
@SebasBarra02 This class should be out of the fuzzy matcher library due to it's not part of it, so please create a new directory and move it.